# hackbio-internship
# RNA-seq is a powerful tool for detecting differentially expressed alleles of transcripts in specific biological processes and can reveal new molecular mechanism of diseases as no reference genome is required. Nowadays, RNA-seq has been wildly used in the study of disease mechanism, clinical diagnosis, and drug development.
## MYCN oncogene amplification is a marker of poor prognosis in patients with neuroblastoma disease. Schramm et al. identified key genes and pathways associated with MYCN by RNA-seq on the SOLiD V4 platform (Schramm et al., 2013).
## RNA-seq has become widely used for investigating disease mechanisms, clinical diagnostics, and drug development.
## In neuroblastoma, MYCN oncogene amplification is linked to poor prognosis. Schramm et al. (2013) used RNA-seq on the SOLiD V4 platform to identify key genes and pathways associated with MYCN amplification.
# Differential gene expression (DGE) analysis is a widely used technique in RNA-sequencing (RNA-seq) data analysis. It is commonly employed in various RNA-seq data processing applications to identify genes that are differentially expressed between two or more sample sets.
## Suryawanshi et al examined the placental transcriptome to analyze gene expression changes across different sections of the placenta. They employed differential expression analysis using DESeq2 to compare gene expression levels between different trimesters of pregnancy and between the fetal and maternal sides of the placenta at term.
## In the research paper “Whole transcriptome profiling of patient-derived xenograft models as a tool to identify both tumor and stromal specific biomarkers” They explored RNA-Seq as a hypothesis-free tool to generate independent tumor and stromal biomarkers and to investigate tumor-stroma interactions by leveraging the human-murine compartment specificity of patient-derived xenografts (PDX).
## Stromal gene expression markers of cancer subtype
## The patient's stroma showed distinct changes typically observed in tumors when comparing TNBC (triple-negative breast cancer) to ER+ (estrogen receptor-positive) samples. This aligns with recent studies suggesting that intrinsic breast cancer subtypes can be identified based on the transcriptome of the tissue adjacent to the tumor.

![differenential expression](https://github.com/user-attachments/assets/cb62fd77-7121-4e6a-a791-c3bf92ac9a8b)


 

 Fig1 : Overlap of gene signatures with genes over-expressed in BTNBC and ER+ luminal-B breast cancers

## These results could indicate intrinsic differences between the patient and mouse stroma. Alternatively, they may be a result of the higher predicted stromal cell purity achieved with our RNA-Seq approach.



# References

### Bradford, J. R., Wappett, M., Beran, G., Logie, A., Delpuech, O., Brown, H., Boros, J., Camp, N. J., McEwen, R., Mazzola, A. M., D’Cruz, C., & Barry, S. T. (2016). Whole transcriptome profiling of patient-derived xenograft models as a tool to identify both tumor and stromal specific biomarkers. Oncotarget, 7(15), 20773–20787. https://doi.org/10.18632/oncotarget.8014
### Yang, X., Kui, L., Tang, M., Li, D., Wei, K., Chen, W., Miao, J., & Dong, Y. (2020). High-Throughput Transcriptome Profiling in Drug and Biomarker Discovery. Frontiers in Genetics, 11. https://doi.org/10.3389/fgene.2020.00019
### Schramm, A., Köster, J., Marschall, T., Martin, M., Schwermer, M., Fielitz, K., Büchel, G., Barann, M., Esser, D., Rosenstiel, P., Rahmann, S., Eggert, A., & Schulte, J. H. (2012). Next‐generation RNA sequencing reveals differential expression of MYCN target genes and suggests the mTOR pathway as a promising therapy target in MYCN‐amplified neuroblastoma. International Journal of Cancer, 132(3). https://doi.org/10.1002/ijc.27787



